tests/test_utility.py

```diff
@@ -1,12 +1,17 @@
-import os
-from contextlib import nullcontext as does_not_raise
+import json
 
 import pytest
 from fastapi import HTTPException
 
 from app.api import utility as util
 
 
+@pytest.fixture
+def tmp_local_nb_nodes_path(tmp_path):
+    """Return a temporary path to a local nodes config JSON file for testing."""
+    return tmp_path / "local_nb_nodes.json"
```

Comment:
I think we can just use `tmp_path` directly in the test and then create the path there, and remove this fixture. This fixture doesn't create any temp files, it just returns a path so we don't save on boilerplate but we lose some readability